testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -286,6 +286,19 @@ def test_parallelizable_transformations():
     with pytest.raises(ValueError):
         FrameAnalysis(u.trajectory).run(backend='multiprocessing')
 
+
+def test_instance_serial_backend(u):
+    serial_backend = backends.BackendSerial(n_workers=1)
+
+    FrameAnalysis(u.trajectory).run(
+        backend=serial_backend,
+        verbose=True,
+        progressbar_kwargs={"leave": True},
+        unsupported_backend=True
+    )
+
+    assert isinstance(serial_backend, backends.BackendSerial)
```

Comment:
To me this test looks a bit self-referential. You are creating a `backends.BackendSerial` and then just checking that said object is indeed `backends.BackendSerial`. Could you please elaborate a bit more what is the intent here? I might be missing something...

Reply:
Thank you for the clarification. To test the part of the code you linked, what I would do is to use `pytest.raises` to make sure an exception is thrown when the backend is not serial.

Reply:
Ah I see, yes that makes sense, I will then adjust the PR accordingly :)

Reply:
@RMeli I now modified the pytest to test the `ValueError` for the case that the backend is not serial. Should I also add before a line before that with the backend serial no Error appears and it only appears if the backend is not serial or is it enough like it is currently with only the case of the backend being not serial?

Reply:
It think it is fine like this, but up to you if you want to add the other case as well (I don't mind, but I don't think it's particularly useful.)

Reply:
Sounds good, then we can keep it like this, since I think some of the other tests cover this indirectly, by running the serial backend for certain tests sucessfully without any Errors appearing.